extensions/wikia/PageShare/PageShareController.class.php

```diff
@@ -38,7 +39,11 @@ private function prepareShareServicesData( $lang ) {
 
 		foreach ( $wgPageShareServices as $service ) {
 			if ( PageShareHelper::isValidShareService( $service, $lang, $isTouchScreen ) ) {
-				$service['icon'] = PageShareHelper::getIcon( $service['name'] );
+				$service['icon'] = DesignSystemHelper::renderSvg(
```

Comment:
Yes, the fallback was removed intentionally. I believe we shouldn't use default sharing icon because it's meaningless. If we would like to add new provider then we should style it properly - no random icons and colors allowed. List of social networks is defined here: https://github.com/Wikia/config/blob/dev/CommonSettings.php#L3514 - wechat is not supported.